src/utils/style-formatter.js

```diff
@@ -13,10 +33,21 @@ export default {
   getStyles(style, Theme) {
     // TODO: use constants for default values?
     let styles = 'width: 100%;height: 100%;font-weight: bold;';
+
     palette = themeResolver.getPalette(Theme);
     styles += formatColorProperty('color', style.fontColor, '#ffffff');
     styles += formatProperty('font-size', style.fontSize ? `${style.fontSize}px` : '12px');
     styles += formatColorProperty('background-color', style.backgroundColor, '#3F8AB3');
+    if (style && style.background && style.background.isUsed) {
+      let bgUrl = style.background.url.qStaticContentUrl.qUrl;
+      bgUrl.replace(/^\.\.\//i, '/');
+      bgUrl = bgUrl.replace(/"/g, '\\"');
+      bgUrl = bgUrl.replace(/'/g, "\\'");
```

Comment:
@Caele I think these checks were copied from the text and image implementation. What we don't have here is the addition of the base path url. Would this be a requirement? Also more in general how do we make sure we are good on a security perspective?